src/data/utility/Util.js

```diff
@@ -225,3 +227,30 @@ export const mergeArraysWithObjects = (a = [], b = [], prop, mergeProp) => {
 
   return updatedArray;
 };
+
+/**
+ * It converts timestamps to UTC based on timezone from user identity.
+ * timestamp, userIdentityId, identities
+ * @param {Number} timestamp datetime timestamp - moment in time
```

Comment:
This should be EPOCH timestamp or are we going to allow any type? @julianideamotive

Reply:
@maartenhemmes By EPOCH do you mean Unix time by seconds or miliseconds? This is how moment lib handles it: http://momentjs.com/docs/#/displaying/unix-timestamp-milliseconds/ 
Let me know which value will we get always from backend, so I'll review the logic here.

Reply:
As discussed with backend team